frontend/src/components/SignForms/SignInForm/styled.tsx

```diff
@@ -0,0 +1,42 @@
+import { styled } from '@mui/styles';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+
+import theme from '../../../themeSettings';
+
+const SignWrapper = styled(Box)({
+  maxWidth: '350px',
+  boxShadow: '0px 0px 10px 5px gray',
+});
+
+const FormBox = styled(Box)({
+  margin: '30px',
+});
+
+const ItemsBox = styled(Box)({
+  margin: '0px !important',
```

Comment:
Do we really need to use important directive in all styled.ts files?

Reply:
yup, because basic material ui styles covering margins/paddings and some styles. Without this in future there may be errors in deploy.